src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java

```diff
@@ -186,6 +186,7 @@ public void sendErrorResponse(HttpServletResponse response, Exception e, int htt
             LOGGER.error(e.toString());
             if (null == errorMsg) errorMsg = e.getMessage();
         }
+        LOGGER.error("Update Recommendation failed due to {}", errorMsg);
```

Comment:
also specify experiment_name and interval_end_time

Reply:
Updated the error log including `experiment_name` and `interval_end_time`